coordinator/internal/userapi/userapi.go

```diff
@@ -46,7 +46,7 @@ type guard interface {
 	// UpdateState advances the state to the given manifest and policies.
 	UpdateState(oldState *stateguard.State, se *seedengine.SeedEngine, manifest []byte, policies [][]byte) (newState *stateguard.State, err error)
 	// ResetState recovers to the latest persisted state, authorizing the recovery seed with the passed func.
-	ResetState(oldState *stateguard.State, a stateguard.SecretSourceAuthorizer) (newState *stateguard.State, err error)
+	ResetState(ctx context.Context, oldState *stateguard.State, a stateguard.SecretSourceAuthorizer) (newState *stateguard.State, err error)
```

Comment:
What about the other interface methods, shouldn't they have a context, too?

Reply:
Ah, comment says this is only used for the authorizer.

Reply:
I think the other methods need a context, too, but wanted to tackle that separately including history. I can add them now, though.